.github/workflows/run_tests.yml

```diff
@@ -3,7 +3,21 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'backend/**'
+      - 'components/**'
+      - 'infrastructure/**'
```

Comment:
As discussed before, I'd like if run_tests run on any change in the project as it's one extra safety net

Reply:
fixed